sbin/spark-daemon.sh

```diff
@@ -82,7 +82,7 @@ spark_rotate_log ()
       num=${SPARK_LOG_MAX_FILES}
     else
       echo "Error: SPARK_LOG_MAX_FILES must be a positive number, but got ${SPARK_LOG_MAX_FILES}"
-      exit -1
+      exit 255
```

Comment:
SC2242 (error): Can only exit with status 0-255. Other data should be written to stdout/stderr.